scripts/taskcluster-android.sh

```diff
@@ -39,6 +39,7 @@ do
     cargo +beta build -p fxa-client-ffi --target ${android_targets[$target]} --release
   mkdir -p fxa-client/$target
   cp target/${android_targets[$target]}/release/libfxa_client.so fxa-client/$target
+  ./libs/bin/patchelf --set-soname libfxa_client.so fxa-client/$target/libfxa_client.so
```

Comment:
A comment that explains why we're doing this and links to the issue would be helpful.